sysl2/sysl/seqs.go

```diff
@@ -101,12 +101,18 @@ func DoConstructSequenceDiagrams(
 					}
 				}
 
+				bbsAll := seqs.TransformBlackboxesToUptos(bbs, seqs.BbApplication)
+				bbsEndpoint := seqs.TransformBlackboxesToUptos(bbs2, seqs.BbEndpointCollection)
+				for k, v := range bbsEndpoint {
+					bbsAll[k] = v
+				}
```

Comment:
```go
m = map[string]*Upto
populateBlackboxMap(m, bbs, uptoType)
populateBlackboxMap(m, bb2, uptoType)
```

Reply:
This is merging the blackboxes from the application and the endpoint collection level.